p2p/docs/connection.md

```diff
@@ -0,0 +1,116 @@
+## MConnection
+
+`MConnection` is a multiplex connection:
+
+__multiplex__ *noun* a system or signal involving simultaneous transmission of
```

Comment:
I don't understand this definition. If I am not wrong any TCP connection satisfies this definition as you can send several messages over it.

Reply:
This is the stock google definition. 

Afaik, TCP is typically considered a single stream - you read and write one stream of messages. This is an abstraction to support reading/writing on multiple streams